src/backends/backend/triton_backend_manager.h

```diff
@@ -143,8 +143,24 @@ class TritonBackendManager {
       const BackendCmdlineConfig& backend_cmdline_config,
       std::shared_ptr<TritonBackend>* backend);
 
+  static TritonBackendManager& Instance();
+  static std::mutex& Mutex()
+  {
+    static std::mutex m;
+    return m;
+  }
+
+  TritonBackendManager() {}
+
+  TritonBackendManager(TritonBackendManager const&) = delete;
```

Comment:
Use DISALLOW_COPY_AND_ASSIGN macro from constants.h.
Put macro at the start of the "private:" section.

Reply:
I removed exposing the Singleton instance altogether. Marking this as resolved.